### PR TITLE
Added New Drinks, Edited Filesystem

### PR DESCRIPTION
**Pull request by SleepyScarecrow**
# Description

Added in 12 new mixed drinks, 2 new bottles to the booze'o'mat, and edited some filesystem stuff that didn't make sense to me. 
The drinks are to give pomjuice and the new aphrodisiacs another purpose, and unfortuantely the bottle'o'cum is just pandering to the masses. The filesystem stuff I changed was simple enough-- there used to be Resources/Prototypes/Floof/ as well as Resources/Prototypes/Floofstation/, I simply merged these both into Resources/Prototypes/Floofstation to make things a bit cleaner.

---

# TODO

- [ ] Pomegranate Foods
- [x] Pomegranate Cocktails

---

<details><summary><h1>Additions</h1></summary>
<p>

(https://cdn.discordapp.com/attachments/1261497448019529790/1263939111392378920/image.png?ex=669c0e87&is=669abd07&hm=e4f0c8f12169961bad739a221c980645bd23540b058b8a78483337f9cbdca635&)
(https://cdn.discordapp.com/attachments/1261497448019529790/1263940347675938919/2024-07-19_15-25-20.mp4?ex=669c0fad&is=669abe2d&hm=3f6cdec1ba6ab7475a46e88f37dc480196764bdfe48f356de230736d575a4baf&)
</p>
</details>

---

# Changelog

:cl: Tilkku
- add: Added a new series of cocktails and new bottles to the booze'o'mat

### CODE REVIEW

**Comment by FoxxoTrystan** [comment]
Did not made a review yet as Linter is unhappy, you should fix that.

Also there no need to put changelogs or code related things, changelogs should only be "in-game"/player related changes, not code or directory, if theres changes please state it in the description, witch you did.

**Comment by SleepyScarecrow** [comment]
made linter happy

**Comment by FoxxoTrystan** [comment]
Also please do not use discord link for media, just paste it directly in github.

File: Resources/Textures/Floof/Objects/Consumable/Drinks/anisettebottle.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "discord: dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/anisettebottle.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

File: Resources/Textures/Floof/Objects/Consumable/Drinks/blueballer.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-3.0",
+    "copyright": "Made by dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/blueballer.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

File: Resources/Textures/Floof/Objects/Consumable/Drinks/bottleocum.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "discord: dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/bottleocum.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

File: Resources/Textures/Floof/Objects/Consumable/Drinks/demonseed.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-3.0",
+    "copyright": "Made by dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/demonseed.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

File: Resources/Textures/Floof/Objects/Consumable/Drinks/doublecreamblaster.rsi/meta.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-3.0",
+    "copyright": "Made by dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/doublecreamblaster.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

File: Resources/Textures/Floof/Objects/Consumable/Drinks/emeraldswinger.rsi/meta.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-3.0",
+    "copyright": "Made by dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/emeraldswinger.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

File: Resources/Textures/Floof/Objects/Consumable/Drinks/mariejulep.rsi/meta.json [hunk 7]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-3.0",
+    "copyright": "Made by dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/mariejulep.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

File: Resources/Textures/Floof/Objects/Consumable/Drinks/orgasmonthebeach.rsi/meta.json [hunk 8]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-3.0",
+    "copyright": "Made by dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/orgasmonthebeach.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

File: Resources/Textures/Floof/Objects/Consumable/Drinks/pompassion.rsi/meta.json [hunk 9]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-3.0",
+    "copyright": "Made by dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/pompassion.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

File: Resources/Textures/Floof/Objects/Consumable/Drinks/redrocket.rsi/meta.json [hunk 10]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-3.0",
+    "copyright": "Made by dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/redrocket.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

File: Resources/Textures/Floof/Objects/Consumable/Drinks/semenbomb.rsi/meta.json [hunk 11]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-3.0",
+    "copyright": "Made by dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/semenbomb.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

File: Resources/Textures/Floof/Objects/Consumable/Drinks/semenhemorrhage.rsi/meta.json [hunk 12]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-3.0",
+    "copyright": "Made by dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/semenhemorrhage.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

File: Resources/Textures/Floof/Objects/Consumable/Drinks/watermelonginjizz.rsi/meta.json [hunk 13]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-3.0",
+    "copyright": "Made by dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/watermelonginjizz.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

File: Resources/Textures/Floof/Objects/Consumable/Drinks/yeolhandy.rsi/meta.json [hunk 14]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-3.0",
+    "copyright": "Made by dakodragon",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/Consumable/Drinks/yeolhandy.rsi/meta.json:8]: Please provide more information
example:

Made by dakodragon (Discord: DiscordID)

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
PR is good, nice change of directory but please make copyright a bit more detailed to find the author if needed, unless the autho wish to not be found.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
DiscordID: dakodragon
 
is incorrect and ID is a string of number.

"Made by dakodragon, DiscordID: 1" example.

**Review by FoxxoTrystan — APPROVED** [review]

**Review by Fansana — APPROVED** [review]

**Pull request merged**